### DISCUSSION TITLE
Feature Request: ON DELETE CASCADE support

### DISCUSSION THREAD

**Discussion by BinaryWarlock** [duckdb/duckdb#10851] [Ideas]
Very useful to ensure foreign key integrity. SQLite supports it and I'm trying to migrate over to DuckDB but this makes it slightly harder.

**Comment by spicychickensauce** [comment]
I'm also trying to migrate from sqlite to duckdb, but this is holding me back.
I was especially intrigued by the [postgres compatibility](https://duckdb.org/docs/stable/sql/dialect/postgresql_compatibility), but cascading deletes were not mentioned there.
